--- /etc/fstab		2006-07-26 16:08:45.000000000 +0100
+++ /root/cd.fstab	2006-07-26 16:18:05.000000000 +0100
@@ -7,3 +7,4 @@
 none                    /proc                   proc    defaults        0 0
 none                    /sys                    sysfs   defaults        0 0
 LABEL=SWAP-sda3         swap                    swap    defaults        0 0
+/dev/cdrom              /media/cdrom            auto    pamconsole,exec,noauto,fscontext=system_u:object_r:removable_t 0 0
